opencga-app/src/main/java/org/opencb/opencga/app/cli/main/custom/CustomStudiesCommandOptions.java

```diff
@@ -65,5 +65,14 @@ public class RunTemplatesCommandOptions {
 
         @Parameter(names = {"--jobTags"}, description = "Job tags.", arity = 1)
         public String jobTags;
+
+        @Parameter(names = {"--jobScheduledStartTime"}, description = "Job scheduled start time.", arity = 1)
```

Comment:
We should open a bug ticket here... this should be in kebab-case, not in camelCase

Reply:
No need for that. That's not generated automatically. It's the CustomOptions so I just changed it

Reply:
Really? So we have an issue, as the other params are also un camel case

Reply:
Oh, you did all of them! :heart: